crates/aptos-telemetry-service/src/prometheus_push_metrics.rs

```diff
@@ -39,7 +42,17 @@ pub async fn handle_metrics_ingest(
     metrics_body: Bytes,
 ) -> anyhow::Result<impl Reply, Rejection> {
     debug!("handling prometheus metrics ingest");
+    let timestamp = Utc::now().timestamp() as usize;
+    push_metrics_to_clients(context, claims, encoding, metrics_body, timestamp).await
+}
 
+pub async fn push_metrics_to_clients(
+    context: Context,
+    claims: Claims,
+    encoding: Option<String>,
+    metrics_body: Bytes,
+    timestamp: usize,
```

Comment:
This timestamp is not used except to cache. This should be passed to `client.post_prometheus_metrics` right?

Reply:
Thanks for pointing it out. Fixed it now.